chatbees/client_models/collection.py

```diff
@@ -277,6 +280,20 @@ def index_ingestion(self, ingestion_id: str):
             ingestion_id=ingestion_id)
         Config.post(url=url, data=req.model_dump_json())
 
+    def delete_ingestion(self, ingestion_type: IngestionType):
```

Comment:
quick validation to reject FILE and WEBSITE?

Reply:
probably no need, ingestion_type.py only defines 
class IngestionType(Enum):     
    CONFLUENCE = 'CONFLUENCE'  
    GDRIVE = 'GDRIVE'          
    NOTION = 'NOTION'